scripts/ai/turns/AITurn.lua

```diff
@@ -591,6 +591,8 @@ function CourseTurn:endTurn(dt)
         if not self.implementsLowered then
             -- have not started lowering implements yet
             self:debug('Turn ending, lowering implements')
+            -- driveStrategy.state can be set to WORKING which is also checked if harvester header can be turned on
+            self.driveStrategy.state = self.driveStrategy.states.WORKING
```

Comment:
It is a major no-no to set a private variable from outside the class. Strategy state must not be changed from a different class.

Reply:
(especially not such an apparently hacky way to turn the cutter on through side effects)